src/components/shared/__tests__/ActionMenu.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { vi } from 'vitest'
+
+import { ActionMenu } from '@/components/shared/ActionMenu'
+import { ActionItem } from '@/types/common.types'
+
+const mockActions: Array<ActionItem> = [
+  {
+    action: vi.fn,
+    label: 'label1',
+  },
+]
+
+describe("Checks that ActionMenu snapshots don't change", () => {
+  it('renders correctly', () => {
+    const inlineClipboard = render(<ActionMenu actions={mockActions} />)
```

Comment:
Should be `actionMenu`. Also @Carminepo2 we should add more test cases

Reply:
fixed name and added other test cases